templates/thresh-config.yml.j2

```diff
@@ -1,6 +1,13 @@
 metricSpoutThreads: 2
 metricSpoutTasks: 2
 
+statsdConfig:
+  host: {{mini_mon_host}}
```

Comment:
This role isn't used only for mini-mon but for all deployments, so mini-mon_host may not be defined. A new variable in defaults/main.yml should be made for this and set to localhost.